WordPress/Classes/Networking/PostServiceRemoteXMLRPC.m

```diff
@@ -164,6 +169,54 @@ - (void)deletePost:(RemotePost *)post
     }
 }
 
+- (void)trashPost:(RemotePost *)post
+          forBlog:(Blog *)blog
+          success:(void (^)(RemotePost *))success
+          failure:(void (^)(NSError *))failure
+{
+    NSParameterAssert([post.postID longLongValue] > 0);
+    NSNumber *postID = post.postID;
+    if ([postID longLongValue] > 0) {
+        NSArray *parameters = [blog getXMLRPCArgsWithExtra:postID];
+
+        WPXMLRPCRequest *deletePostRequest = [self.api XMLRPCRequestWithMethod:@"wp.deletePost" parameters:parameters];
```

Comment:
I think that last bit would be more like after respecting `:` alignment : 
```

        WPXMLRPCRequestOperation *getOperation = [self.api XMLRPCRequestOperationWithRequest:getPostRequest
                                                                                     success:^(AFHTTPRequestOperation *operation, id responseObject) {
                                                                                         if (success) {
                                                                                             // The post was trashed but not yet deleted.
                                                                                             RemotePost *post = [self remotePostFromXMLRPCDictionary:responseObject];
                                                                                             success(post);
                                                                                         }
                                                                                     } failure:^(AFHTTPRequestOperation *operation, NSError *error) {
                                                                                         if (error.code == 404) {
                                                                                             // The post was deleted.
                                                                                             if (success) {
                                                                                                 success(post);
                                                                                             }
                                                                                         }
                                                                                     }];
```

The 80 char limit is a guideline.  While I agree in principle that it helps readibility its feeling like we're trying to force it a bit here for not a lot of gain.